resources/css/pages/worldoverview.css

```diff
@@ -0,0 +1,72 @@
+@import url('../components/colors.css');
```

Comment:
deze lijn kan verwijderd als die gemerged wordt met develop omdat `colors.css` nu geimporteerd wordt in `app.css`